### PR TITLE
Jr reader photos

### PR DESCRIPTION
**Pull request by jreynolds13**
- New page template for reader photos submission form page
- Page copy is queried from Wordpress content 😄
- Fancy slider for example photos with bylines
-- New ACF field group and field for taxonomy on attachments
-- Slider queries photos with the tax 'reader-submitted-example'
- Grav form will need to be ported over to production, ID in code may need to be updated there
- This work corresponds with new Zap for Grav form to auto-generate Google Doc from each submission
- Some global form styling updates are included here

### CODE REVIEW

File: src/assets/js/sliders.js [hunk 1]
```diff
@@ -134,3 +134,16 @@ function updateHeroImage(slick, currentSlide, force = false) {
...
+
+// Reader submitted examples slider
+$(function() {
+  $('.reader-submit__example-photos__slider').slick({
+    infinite: true,
+    swipeToSlide: true,
+    slidesToScroll: 1,
+    autoplay: false,
+    prevArrow: '<button type="button" class="slick-prev" data-ga-event=\'["Interaction", "Instagram-slider", "Previous"]\'><i class="far fa-angle-left fa-lg"></i></button>',
+    nextArrow: '<button type="button" class="slick-next" data-ga-event=\'["Interaction", "Instagram-slider", "Next"]\'><i class="far fa-angle-right fa-lg"></i></button>',
```
Comment (strikemike2k) [inline src/assets/js/sliders.js]: Let's update these GA Event names from Instagram Slider to something more relevant to this slider.

File: src/assets/scss/custom/_usefuls.scss [hunk 2]
```diff
@@ -67,6 +67,16 @@ $site-width: rem-calc(1800);
...
+// @mixin aspect-ratio-slider {
+//   .slick-slide {
+//     padding-bottom: aspect-ratio($width, $height);
+//     height: 0;
+//     overflow: hidden;
+//     position: relative;
+//     border-radius: 6px;
+//   }
+// }
+
```
Comment (strikemike2k) [inline src/assets/scss/custom/_usefuls.scss]: Not needed?

File: page-templates/reader-submit.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                  </cite>
+                </div>
+              </div>
+            <?php } ?>
+        </div>
+      </div>
+    </div>
+    <div class="reader-submit__section reader-submit__form">
+      <h5>Complete the form below to submit your photo for consideration</h5>
+      <?php gravity_form(4, false, false) ?>
```
Comment (strikemike2k) [inline page-templates/reader-submit.php]: I wonder if we can use environment variables to define the form ID per environment.

Comment (jreynolds13) [reply page-templates/reader-submit.php]: Taylor and I were just discussing exactly that.

Comment (jreynolds13) [reply page-templates/reader-submit.php]: Or rather, we were discussing setting up an ACF for pages to input a form ID, and then querying that ACF field value.

Comment (jreynolds13) [reply page-templates/reader-submit.php]: Environment variables could be good, although ideally we'd like to sync all of our local/staging databases up with production (or vice-versa) so we have consistent IDs.

File: src/assets/scss/custom/_usefuls.scss (hunk 2 above)
Comment (jreynolds13) [reply src/assets/scss/custom/_usefuls.scss]: Good catch

File: src/assets/js/sliders.js (hunk 1 above)
Comment (jreynolds13) [reply src/assets/js/sliders.js]: Good catch

**Pull request merged**